frontend/src/common/models/IUser.ts

```diff
@@ -0,0 +1,5 @@
+export default interface IUser {
```

Comment:
I think better to create file **user.ts** and place there named export with initial IUser interface.